src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const arend = days * 40;
```

Comment:
Consider renaming the variable 'arend' to 'rent' or 'rentalCost' for better clarity.